activiti-spring-boot-starter/src/test/resources/processes/variable-mapping-extensions.json

```diff
@@ -0,0 +1,47 @@
+{
+  "id":"connectorVarMapping",
+  "name":"myProcess.bpmn",
+  "extensions": {
+    "properties": {
+      "age-id": {
+        "id": "age-id",
+        "name": "age",
+        "type": "integer",
+        "value": 20
+      },
+      "name-id": {
+        "id": "name-id",
+        "name": "name",
+        "type": "string",
+        "required": true,
+        "value": "inName"
+      },
+      "nick-name-id": {
+        "id": "nick-name-id",
+        "name": "nickName",
+        "type": "string",
+        "required": false,
+        "value": "testName"
+      },
+      "input-unmapped-variable-name-id": {
+        "id": "input-unmapped-variable-name-id",
+        "name": "input-unmapped-variable-name",
+        "type": "string",
+        "required": false,
+        "value": "inTest"
+      }
+    },
+    "variablesMappings": {
```

Comment:
The variablesMappings section of this json it's not anymore valid. @popovicsandras can you add here the new format we agreed together with @salaboy ?

Reply:
This should be the new format of the variablesMappings section:
```
"mappings": {
    "Task_1hkl8bn": {
      "input": {
        "726478cc-91ac-40bb-8f3c-f0782ab81af0": {
          "type": "variable",
          "value": "5e4947c2-d263-4b86-ab98-5627ff5f461a"
        },
        "33ec3b54-90ff-4a97-91be-178976a4dd26": {
          "type": "value",
          // not process variable, comes from the connector configuration, BE magic
          "value": "${host}"
        }
      }
    }
  }
```

Reply:
@ffazzini @magemello @popovicsandras `input` here (singular) is conflict with `inputs` (plural) suggested by @ffazzini for [connector definitions](https://github.com/Activiti/Activiti/pull/2227/files#r251249714). It should be plural as well I guess.

Reply:
It was plural always. I don't know where this information came from, the connector editor created a definition with the plurals (inputs, outputs) from the very beginning.